doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -377,3 +377,78 @@ settings to ensure that all IOs are issued through SPDK.::
 
 Otherwise, the current implementation will populate the SPDK map files with
 kernel file system symbols and will use the kernel driver to issue DB/WAL IO.
+
+Minimum Allocation Size
+========================
+
+There is a configured minimum amount of storage that BlueStore will allocate on
+an OSD.  In practice, this is the least amount of capacity that a RADOS object
+can consume.  The value of :confval:`bluestore_min_alloc_size` is derived from the
+value of :confval:`bluestore_min_alloc_size_hdd` or :confval:`bluestore_min_alloc_size_ssd`
+depending on the OSD's ``rotational`` attribute.  This means that when an OSD
+is created on an HDD, BlueStore will be initialized with the current value
+of :confval:`bluestore_min_alloc_size_hdd`, and SSD OSDs (including NVMe devices)
+with the value of :confval:`bluestore_min_alloc_size_ssd`.
+
+Note that this BlueStore attribute takes effect *only* at OSD creation; if
+changed later, a given OSD's behavior will not change unless / until it is
+destroyed and redeployed.
+
+Through the Mimic release, the default values were 64KB and 16KB for rotational
+(HDD) and non-rotational (SSD) media respectively.  Octopus and later releases
+default to a value of 4KB for all media types.
```

Comment:
This was already merged but I just noticed, but I believe min_alloc_size_hdd was changed in Pacific, it was still 64k in Octopus.

Reply:
Hmmm.  Actually we're both right.  Looks like there was a bit of back and forth.  I'd found

https://docs.ceph.com/en/latest/releases/octopus/

mentioning https://github.com/ceph/ceph/pull/30698

but missed

https://github.com/ceph/ceph/pull/32809

reverting _hdd shortly thereafter.

I'll put in a new PR. Thanks for catching this.